Add gzip fallback

kylebarron opened the pull request:
Closes #9 

Adds `gzip_fallback` argument to compress with gzip when `br` is not in the Accept-Encoding headers.

fullonic commented:
Great job @kylebarron . Thanks a lot for your contribution. 
One last thing, may you just update de README file to reflect the new changes?

kylebarron commented:
I modified the README to document the new `gzip_fallback` argument. I also changed `brotli-middleware` to `brotli-asgi` in the README since on PyPI [`brotli-asgi`](https://pypi.org/project/brotli-asgi/) is newer than [`brotli-middleware`](https://pypi.org/project/brotli-middleware/) (should also resolve #8).

kylebarron commented:
Thanks! Would you be interested in pushing a new release to PyPI?

fullonic commented:
Great. Thanks for it. I will soon update as well the pip repo version.
